packages/contractkit/src/base.ts

```diff
@@ -24,3 +24,9 @@ export const AllContracts = Object.keys(CeloContract).map(
 ) as CeloContract[]
 
 export const NULL_ADDRESS = '0x0000000000000000000000000000000000000000' as Address
+
+export enum Roles {
```

Comment:
just notice this is in `base.ts`. Let's move it to `LockedGold`, since it's the only place we have the right context.